src/index.html

```diff
@@ -11,22 +11,38 @@
       content="ie=edge"
     />
     <title>Document</title>
+
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
+      <label>
+        <input
+          class="search-bar search-bar__big"
```

Comment:
It looks like you have two classes assigned to the same element. If 'search-bar' is a block, it should not be a child of itself ('search-bar__big'). Please consider revising your class naming to adhere to BEM methodology.